var/spack/repos/builtin/packages/mysql/package.py

```diff
@@ -135,27 +135,28 @@ def _fix_dtrace_shebang(self, spack_env):
         copy(dtrace, dtrace_copy)
         filter_file(
             '^#!/usr/bin/python',
-            '#!/usr/bin/env python',
+            '#!/usr/bin/env {0}'.format(
+                os.path.basename(self.spec['python'].command)),
             dtrace_copy
         )
         # To have our own copy of dtrace in PATH, we need to
         # prepend to PATH the temporary folder where it resides.
-        spack_env.prepend_path('PATH', dtrace_copy_path)
+        build_env.prepend_path('PATH', dtrace_copy_path)
 
     @run_before('cmake')
     def _maybe_fix_dtrace_shebang(self):
         if 'python' in self.spec.flat_dependencies() and \
            self.spec.satisfies('@:7.99.99'):
-            self._fix_dtrace_shebang(spack_env)
+            self._fix_dtrace_shebang(build_env)
```

Comment:
Where is this variable actually set?

Reply:
https://github.com/spack/spack/blob/develop/lib/spack/spack/build_environment.py#L676 Magic :stuck_out_tongue: